plugins/modules/system/keyring.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Alexander Hussey <ahussey@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+Ansible Module - community.general.keyring
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: keyring
```

Comment:
```suggestion
module: keyring
version_added: 5.1.0
```

possibly 5.1.0, @felixfontein  knows better

Reply:
The 5.1.0 release will be on Tuesday, this will be pretty tight. If we can finish this until Monday 5.1.0 fits, otherwise it will be probably 5.2.0.

Reply:
I'll aim to complete these changes within the next 24 hours, but it is a tight turn around, so 5.2.0 might be more realistic